packages/client/src/components/PathModule/PathModule.module.css

```diff
@@ -0,0 +1,22 @@
+.completed {
+    fill:  #D7FAF3;  /* path */
+    stroke: #0DD2AB; /* path */
+    color: #0DD2AB;   /* font color*/
+}
+
+.open {
+    fill:  #EEEBF6;
+    stroke: #1F00C5;
+    color: #1F00C5;
+}
+
+.locked {
+    fill:  #FFFFFF;
+    stroke: #C4BEE0;
+    color: #C4BEE0;
+}
+
+.checkmark {
+    top: -2px;
+    right: 16px;
+}
```

Comment:
Can we please use `@apply ...` tailwind stuff here? We don't ever want to be referencing hex codes where possible ever. Makes it very hard to update colors etc across the project.

Reply:
1. Three color defined in figma are not defined in tailwind.config.js, so picked similar color. Please refer to the comments in PathModule.module.css for details.
2. right: 16px is kept. Because I did not find corresponding tailwind definition.